experimental/cloudstrapper/README.md

```diff
@@ -0,0 +1,178 @@
+
+  - Login to Cloustrapper node via SSH to start Build, Control Plane and Data Plane rollouts
+
+  - Locate ~/code/mantle/magma-on-aws/playbooks/vars/secrets.yaml file and fill out Secrets
+    section and save it in WORK_DIR on Cloudstrapper. Optionally, change other values if required.
+
+## 3. Build
+
+  The build- playbooks provision, configure and initiate the build process before posting 
+  the artifacts on identified repositories on successful build.
+
+  - Create build elements: Provision, Configure and Init. 
+     
+  - Before beginning Build process, check variables to ensure deployment is customized.
+    build.yaml : 
+      - buildMagmaVersion indicates which version of Magma to build (v1.3, v1.4 etc)
+      - buildOrc8rLabel indicates what label the images would have
+      - buildHelmRepo indicates which github repo will hold Helm charts
+
+      - buildAwsRegion indicates which region will host the build instance. 
+      - buildAwsAz indicates an AZ within the region specified above
+
+  - build-provision: Setup build instance using default security group, Bootkey and Ubuntu with
+    t2.xlarge. Optionally, Provision a AGW compliant image (Debian 4909 or Ubuntu 20.04) 
+    ```
+    ansible-playbook build-provision.yaml
+    ```
+
+  - build-configure: Configure build instance by setting up necessary parameters and reading from
+    dynamic inventory. The build node was provisioned with the tag Name:buildOrc8r in this example.
+    ```
+    ansible-playbook build-configure.yaml -i ~/magma-experimental/files/common_instance_aws_ec2.yaml -e "buildnode=tag_Name_buildOrc8r" -e "ansible_python_interpreter=/usr/bin/python3"
+    ```
+
+  - build-ami-configure: Configure AMI for AGW by configuring base AMI image with AGW packages and
+    building OVS.
+    *TODO: Remove existing snapshot and create new snapshot*
+    ```
+    ansible-playbook build-ami-configure.yaml -e '@vars/main.yaml' -i files/build_instance_aws_ec2.yaml -e "buildnode=tag_Name_ec2MagmaBuild" -u admin
+    ```
+
+  - Result: Build instance created, images and helm charts published. AGW AMI created.
+
+## 4. Control Plane/Cloud Services
+
+  The control- roles deploy and configure orc8r in a target region.
+
+  - Create control plane elements: Provision, Configure and Init
+    Observe the variables set in cluster.yaml
+
+    Make any custom changes to main.tf here before initializing. If you would like to persist changes
+    across re-installs, make changes to the main.tf.j2 Jinja2 template file directly so that the custom
+    configuration be used across every terraform init.
+
+  - Requires: secrets.yaml in the dirInventory folder. Use the sample file in roles/vars/secrets.yaml
+  - Orchestrator : Deploy orchestrator 
+  ```
+    ansible-playbook orc8r.yaml [ --skip-tags deploy-orc8r ]
+  ```
+
+  Note: First time installs might want to skip using Terraform from within Ansible to make sure the
```

Comment:
Can you also suggest if the user does skip the deploy through this process, where they might have to go and manually run `terraform init && terraform apply` and steps they might have to run to deploy orc8r or provide a pointer to the appropriate section.

Reply:
Added new text to indicate what commands to run when deploy-orc8r tag is skipped